src/widget/city.c

```diff
@@ -368,11 +368,21 @@ int widget_city_has_input(void)
 
 void widget_city_handle_mouse(const mouse *m)
 {
-    scroll_map(m);
```

Comment:
Does the map still scroll properly for touch mode now?

Reply:
I'll have a look at this.

Reply:
I can confirm that the decaying scrolling doesn't work on the Vita with this PR.

Reply:
I tested after the decay scroll fix commit and it seems to be working fine 👍